src/main/java/com/feedzai/commons/sql/abstraction/engine/configuration/PdbProperties.java

```diff
@@ -144,6 +145,11 @@
      * Property that indicates the lobs should be compressed. This depends on the database implementation.
      */
     public static final String COMPRESS_LOBS = "pdb.compress_lobs";
+    /**
+     * Property that indicates if LOB data caching should be disabled, to avoid consuming too much memory and/or disk
+     * space in the DB server. This depends on the database implementation.
+     */
```

Comment:
Suggest you mention this is applicable only to Oracle.

Reply:
I see what you mean...
I added the "This depends on the database implementation" note, just like the _compress_lobs_ property has.

Both properties currently do something only on Oracle, but if an issue with LOB data increasing disk/memory usage is found in other DB implementations, this property is generic enough to also apply as a toggle for custom solutions specific for those implementations.